pkg/Makefile.am

```diff
@@ -4,12 +4,21 @@ pkg_TESTS = \
 
 TESTS += $(pkg_TESTS)
 
+metainfo_in = \
+	pkg/sosreport/org.cockpit-project.cockpit-sosreport.metainfo.xml \
+	pkg/kdump/org.cockpit-project.cockpit-kdump.metainfo.xml \
+	pkg/selinux/org.cockpit-project.cockpit-selinux.metainfo.xml \
+	pkg/storaged/org.cockpit-project.cockpit-storaged.metainfo.xml \
+	$(NULL)
+EXTRA_DIST += $(metainfo_in)
+
+dist/%.metainfo.xml: pkg/%.metainfo.xml
```

Comment:
I feel like we just finished saying that we want `dist/` to be only for webpack output...

Reply:
Ack, moved built files to pkg/*/.